bindings/python/src/layers.rs

```diff
@@ -75,7 +54,7 @@ impl RetryLayer {
         jitter: bool,
         max_delay: Option<f64>,
         min_delay: Option<f64>,
-    ) -> PyResult<Self> {
+    ) -> PyResult<(Self, Layer)> {
```

Comment:
I'd prefer to use [PyClassInitializer](https://docs.rs/pyo3/latest/pyo3/pyclass_init/struct.PyClassInitializer.html) instead of returning tuple because the intention is clearer.

Reply:
Cool, doesn't know about this, let me have a try.

Reply:
Updated.